### PR TITLE
add test Wallet has the same tip as network/information

### PR DESCRIPTION
**Pull request by piotr-iohk**
# Issue Number

#819

# Overview

- [ ] add test checking that wallet has the same tip as network/information endpoint

# Comments

### CODE REVIEW

File: lib/core-integration/src/Test/Integration/Scenario/API/Network.hs [hunk 1]
```diff
@@ -25,33 +23,51 @@ import Test.Integration.Framework.DSL
...
                     (Quantity 0, Quantity $ fromIntegral $ sl + 1)
                 ]
+    it "NETWORK2 - Wallet has the same tip as network/information" $ \ctx -> do
+        let getNetworkInfo = request @ApiNetworkInformation ctx networkInfoEp Default Empty
+        w <- emptyWallet ctx
+        eventually $ do
+            sync <- getNetworkInfo
+            verify sync [ expectFieldEqual DSL.syncProgress Ready ]
+        r <- getNetworkInfo
+        let (Right epochNum) = fmap (epochNumber . tip) (snd r)
```
Comment (KtorZ) [inline lib/core-integration/src/Test/Integration/Scenario/API/Network.hs]:
```suggestion
let epochNum = getFromResponse (#tip . #epochNumber) 
```

File: lib/core-integration/src/Test/Integration/Scenario/API/Network.hs [hunk 2]
```diff
@@ -25,33 +23,51 @@ import Test.Integration.Framework.DSL
...
+        let (Right slotNum) = fmap (slotNumber . tip) (snd r)
+        let (Right blockHeight) = fmap (height . tip) (snd r)
+
+        expectEventually' ctx state Ready w
+        expectEventually' ctx (#tip . #epochNumber) epochNum w
+        expectEventually' ctx (#tip . #slotNumber) slotNum  w
+        expectEventually' ctx (#tip . #height) blockHeight w
```
Comment (KtorZ) [inline lib/core-integration/src/Test/Integration/Scenario/API/Network.hs:70]: :+1:

**Review by KtorZ — APPROVED** [review]

File: lib/core-integration/src/Test/Integration/Scenario/API/Network.hs (hunk 1 above)
Comment (piotr-iohk) [reply lib/core-integration/src/Test/Integration/Scenario/API/Network.hs]: updated!

**Review by paweljakubas — APPROVED** [review]
LGTM

**Pull request merged**